src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -12,23 +18,48 @@
 
 public class SelectTestFigureOptionListener implements ActionListener {
 
-	private DriverManager driverManager;
+    private DriverManager driverManager;
 
-	public SelectTestFigureOptionListener(DriverManager driverManager) {
-		this.driverManager = driverManager;
-	}
+    public SelectTestFigureOptionListener(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
 
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.